BeeKit/UI/UIColorExtension.swift

```diff
@@ -12,20 +12,21 @@ import UIKit
 extension UIColor {
     
     public struct Beeminder {
-        public static let green = UIColor(red: 81.0/255.0,
-                                          green: 163.0/255.0,
-                                          blue: 81.0/255.0,
-                                          alpha: 1)
-        
-        public static let blue: UIColor = .systemBlue
-        public static let orange: UIColor = .systemOrange
         public static let red: UIColor = .systemRed
         
-        public static let gray = UIColor(white: 0.7, alpha: 1.0)
+        public static let gray = UIColor.systemGray
```

Comment:
I believe this is a somewhat darker color than the previous one (more like `white: 0.5`). Was that deliberate?

Reply:
> The [system gray one will adapt to the device's environment](https://developer.apple.com/documentation/uikit/uicolor/3173143-systemgray)

I agree this is true in abstract, and for most system colors. It turns out `systemGray` is the same in both dark mode and light mode. I'm not convinced having this universal gray really makes sense (e.g. perhaps backgrounds should be closer to the background color than borders are), but that seems like a problem for another day.